src/beneuro_data/cli.py

```diff
@@ -678,6 +742,119 @@ def upload_session(
     return True
 
 
+@app.command()
+def up(
+    session_name: Annotated[
```

Comment:
Now this is not necessarily a session name, so please change the argument name and the help to indicate that. `session_or_animal_name` would work.

Reply:
good point